src/modules/job-manager/plugins/limit-duration.c

```diff
@@ -200,7 +200,7 @@ static int check_limit (struct limit_duration *ctx,
         && limit != DURATION_UNLIMITED
```

Comment:
Commit message:
Using the term "average" when talking about numerical precision is a bit confusing.  I assume you meant something like "typical"?